internal/database/unit_test.go

```diff
@@ -394,14 +394,7 @@ func machineInfoUpdatesWork(t *testing.T, db database.Database) {
 		Group:       &fakeGroup,
 	}
 
-	err = db.UpdateMachine(fakeChange)
-
-	// TODO: remove this
-	// skip errors, as inmemory doesn't currently implement update
-	//assert.NoError(t, err)
-	if err != nil {
-		t.Skipf("Skipping with error, was %v", err)
-	}
+	_ = db.UpdateMachine(fakeChange)
```

Comment:
Don't we want to assert that no error is returned here, rather than discard the return value?